backend/src/main/java/org/example/backend/service/ShopService.java

```diff
@@ -42,4 +44,20 @@ public void deleteList(String id) {
     public Optional<ShoppingList> getListById(String id) {
         return listRepo.findById(id);
     }
+
+    public long getTotalItems(String id) {
+        return listRepo.findById(id)
+                .map(list -> list.item().size())
+                .orElse(0);
```

Comment:
"0L" wie du es auch in Zeile 57 gemacht hast

Reply:
Das ist leider nicht möglich, da die size-Funktion mir einen integer auswirft und deshalb kann ich hier kein Long nehmen.

Reply:
Ich habe die Funktion so angepasst um ein Long zu erhalten: .map(list -> (long) list.item().size())
                .orElse(0L);